ngraph/core/reference/include/ngraph/runtime/reference/softplus.hpp

```diff
@@ -16,7 +16,7 @@ namespace ngraph
             template <typename T>
             void softplus(const T* arg, T* out, size_t count)
             {
-                const T threshold = static_cast<T>(std::log(std::numeric_limits<T>::max()));
+                const T threshold = 20.0;
```

Comment:
Why do you use hardcoded value (Why 20?)?

Reply:
@ilyachur current implementation is not aligned with spec. threshold is a range that we can choose from, 20 is chosen for FP32 based on experiment, to align to the spec and cpu-plugin implementation.  for FP16, it is also working that no overflow will happen as calculation is based on FP32 and converted to FP16 at final stage.  So either we change the spec or change the code here to get it aligned.

Reply:
revert back ..